src/utils/handleAndSignImages.ts

```diff
@@ -0,0 +1,32 @@
+import isSVGImage from '@/utils/isSVG';
```

Comment:
This util file is extracted and improved from [this file](https://github.com/rainbow-me/rainbow/blob/develop/src/parsers/uniqueTokens.js#L48-L68), please review against that to ensure no regressions.